### ISSUE TITLE
Default Title and Default Caption

### ISSUE THREAD

**Issue by giucu91** [MachoThemes/modula-lite#266]
We should remove this settings if we get image title and caption from media library.

**Comment by razvanaldea89** [comment]
Yeah, but with these options users can select which goes which. Title can have Image title or Image description and Caption can have Image title, Image description or Image caption. This can be very usefull in some situations. For example, if users want to import a gallery, or use images from another place in the website and they want to invert the title/caption/description.

**Comment by cristianraiber** [comment]
@razvanaldea89 - while you do make a valid point, we're reinveting the wheel here :) WP's native media gallery offers all these fields each with its own value, for a reason. Allowing the users to mix up these values isn't really going to do them any good. If they still want to do it, they can just copy/paste the values from one field to another, manually.

The biggest pain point with these settings right now, is that they're not being taken into account properly and they're overwriting the importing of: title, captions, descriptions and alt texts from WP's Media Gallery. 

**TL;DR** - we'll revisit this down the road if anyone complains. Right now, the use case you've described above, while valid, isn't being used by any of our users :)

**Comment by andyluak** [comment]
The settings don't appear anymore and they have been commented out .

**Comment by andrei-l-cristea** [comment]
Tested this also and the options have been properly removed.
